tfhe/benches/integer/bench.rs

```diff
@@ -58,27 +55,48 @@ impl Default for ParamsAndNumBlocksIter {
             Err(_) => false,
         };
 
-        let bit_sizes = if is_fast_bench {
+        EnvConfig {
+            is_multi_bit,
+            is_fast_bench,
+        }
+    }
+}
+
+fn bit_sizes(env_config: &EnvConfig) -> Vec<usize> {
+    if env_config.is_multi_bit {
+        if env_config.is_fast_bench {
             FAST_BENCH_BIT_SIZES.to_vec()
-        } else {
+        } else if cfg!(feature = "gpu") {
             BENCH_BIT_SIZES.to_vec()
-        };
+        } else {
+            vec![8, 16, 32, 40, 64]
+        }
+    } else if env_config.is_fast_bench {
+        FAST_BENCH_BIT_SIZES.to_vec()
+    } else {
+        BENCH_BIT_SIZES.to_vec()
+    }
+}
```

Comment:
This should actually be a method of EnvConfig

Reply:
indeed